src/test/scala/is/hail/methods/KeyTableSuite.scala

```diff
@@ -145,17 +145,17 @@ class KeyTableSuite extends SparkSuite {
     assert(ktLeftJoin.count == ktLeft.count &&
       ktLeftJoin.nKeys == nExpectedKeys &&
       ktLeftJoin.nColumns == nExpectedColumns &&
-      ktLeftJoin.filter { a =>
+      ktLeftJoin.copy(rdd = ktLeftJoin.rdd.filter { a =>
```

Comment:
why this change?

Reply:
I got rid of KeyTable.filter(annotation => boolean) method since this was the only place it was used.